.github/workflows/temsimulator.yml

```diff
@@ -0,0 +1,48 @@
+name: "Building TEM simulator container"
+
+on:
+  push:
+    branches: [ master, temsim-container ]
+
+  pull_request:
+    branch: [ master, temsim-container ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
is this some cloud "container"?

Reply:
so here we are asking GitHub Actions to build our container as if we were on a Linux machine whose OS is Ubuntu.